nodeup/pkg/model/protokube.go

```diff
@@ -108,12 +108,27 @@ func (t *ProtokubeBuilder) buildSystemdService() (*nodetasks.Service, error) {
 		"-v", "/:/rootfs/",
 		"-v", "/var/run/dbus:/var/run/dbus",
 		"-v", "/run/systemd:/run/systemd",
-		"--net=host", "--privileged",
+	}
+
+	// add kubectl only if a master
+	// path changes depending on distro, and always mount it on /opt/kops/bin
+	// kubectl is downloaded an installed by other tasks
+	if t.IsMaster {
+		dockerArgs = append(dockerArgs, []string{
+			"-v", t.KubectlPath() + ":/opt/kops/bin:ro",
```

Comment:
So docker can actually bind mount a file, which is nice because it avoids dragging in everything in the bin directory.  Not sure whether I care, given we mount `/` -> `/rootfs` above...

A potential gotcha is that we have to be careful about shared libraries, but thankfully kubectl is statically linked.

Reply:
Right, but we have a chance that protokube will start before kubectl is installed ... async tasks ...

Reply:
Ah good point, and then we would hit the docker bug where it assumes a directory.

The tasks are technically ordered if we tell nodeup about our dependencies, but I do like your way better.